### PR TITLE
#942: fix up path computation for project-fork list

### PR DESCRIPTION
**Pull request by choyrim**
fix for #942 - path computation wasn't working.

### CODE REVIEW

**Comment by max-wittig** [comment]
@choyrim But don't you delete the `list` method here and replace it with the `create` method?

**Comment by choyrim** [comment]
> @choyrim But don't you delete the `list` method here and replace it with the `create` method?

@max-wittig yes that is the end result. I originally just tried to fix the list method but that caused the create to fail. So I noticed that the `ProjectPipelineManager` had resolved a similar path plurality (pipeline vs pipelines) issue and imitated the approach.

File: gitlab/v4/objects.py [hunk 1]
```diff
@@ -2069,27 +2069,24 @@ class ProjectForkManager(CreateMixin, ListMixin, RESTManager):
...
-        """
+            GitlabCreateError: If the server cannot perform the request
 
-        path = self._compute_path("/projects/%(project_id)s/forks")
-        return ListMixin.list(self, path=path, **kwargs)
+        Returns:
+            RESTObject: A new instance of the managed object class build with
+                the data sent by the server
+        """
+        path = self.path[:-1]  # drop the 's'
```
Comment (max-wittig) [inline gitlab/v4/objects.py]: Maybe we could use the real path instead to avoid confusion.

```
/projects/%(project_id)s/fork
```

**Comment by max-wittig** [comment]
Ah I see. Makes sense! Thank you!

File: gitlab/v4/objects.py (hunk 1 above)
Comment (choyrim) [reply gitlab/v4/objects.py]: I am just trusting that the person who did the pipeline manager knew what he/she was doing. I suspect that if I replace the path as above, it won't substitute in for `%(project_id)s` which was the cause for the 404 in the earlier `ProjectForkManager.list` method implementation.

Comment (max-wittig) [reply gitlab/v4/objects.py]: ah you're right. Sorry.

**Pull request merged**